### DISCUSSION TITLE
Spinal Tap Bug: Kicked for Abnormal NPC Buff

### DISCUSSION THREAD

**Discussion by ScuttleBaka** [Pryaxis/TShock#2137] [Q&A] [answered]
hello i was playing on a public-ish server that had tshock on it and when i used the spinal tap (new 1,4,1 whip) it kicked me for adding buff to NPC abnormally, will this be fixed and is there any fix for it

![Screenshot_726](https://user-images.githubusercontent.com/68253667/96339647-0913d200-1064-11eb-9add-36474ba5fb80.png)

**Comment by Quinci135** [comment, marked as answer]
This is already fixed. Most likely they do not have the few changes to bouncer (tshock's anti-cheat) that include 1.4.1 buffs. Though by now, they should have updated their tshock thus this is resolved. The differences between 1.4.0.5 and 1.4.1 are so small that 1.4.0.5 mostly works with 1.4.1.
